src/utils/validateIdentifierInputs.test.ts

```diff
@@ -307,3 +313,88 @@ describe("validateIdentifierInputs", () => {
     ).toThrow("Enter or scan a valid code.");
   });
 });
+
+describe("tests for validateIdentifierInputs for PAYMENT_QR identifier", () => {
+  it("should return true if it contains valid SGQR payload with NETS", () => {
+    expect.assertions(1);
+    expect(
+      validateIdentifierInputs([
+        {
+          label: "Payment QR",
+          scanButtonType: "QR",
+          validationRegex: "[\\s\\S]*",
+          textInputType: "PAYMENT_QR",
+          value:
+            "00020101021102164761360000000*1704155123456789123451110123456789012153123456789012341531250003440001034450003445311000126330015SG.COM.DASH.WWW0110000005550127670014A00000076200010120COM.LQDPALLIANCE.WWW021512345678901234503020028660011SG.COM.OCBC0147OCBCP2P629A358D-ECE7-4554-AD56-EBD12D84CA7E4F7329500006SG.EZI013603600006-76bb-4a5a-aa1a-fbcb64d6ecf530850013SG.COM.EZLINK01201234567890123456-1230204SGQR0324A123456,B123456,C12345670404A23X31260008COM.GRAB0110A93FO3230Q32390007COM.DBS011012345678900210123456789033900011SG.COM.NETS01230201401832831128823590002150001118703240000308885872010901199084E5DC3D834430017COM.QQ.WEIXIN.PAY011012345678900204123435660010SG.COM.UOB014845D233507F5E8C306E3871A4E9FACA601A80C114B5645E5D36840009SG.PAYNOW010120216+621234567890123030100408209912310525123456789012345678901234537270009SG.AIRPAY0110A11BC0000X51860007SG.SGQR0112180307510317020701.0003030608100604020205031380609Counter010708201804075204581453037025802SG5916FOOD XYZ PTE LTD6009SINGAPORE6106081006626001081234567R0607876543050330015SG.COM.DASH.WWW0110000005550263045E8D",
```

Comment:
can we try using the `testConstants` from the `payment-qr-parser` lib?

Reply:
Have changed to use the test constants in the library in [7b3a692](https://github.com/rationally-app/mobile-application/pull/537/commits/7b3a692c4d8788a42de5d70f62fc6a946d25975d)!